crates/bevy_state/src/app.rs

```diff
@@ -73,7 +73,9 @@ impl AppExtStates for SubApp {
             self.init_resource::<State<S>>()
                 .init_resource::<NextState<S>>()
                 .add_event::<StateTransitionEvent<S>>();
-            let schedule = self.get_schedule_mut(StateTransition).unwrap();
+            let schedule = self.get_schedule_mut(StateTransition).expect(
+                "The `StateTransition` schedule is missing. Have you added the `StatesPlugin` to your App?"
```

Comment:
```suggestion
                "The `StateTransition` schedule is missing. Did you forget to add the `StatesPlugin` to your App?"
```
Minor suggestion on the wording here to match other similar errors.